metacat-schema-converters/src/main/java/com/netflix/metacat/shema/converters/HiveMetacatConverters.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.netflix.metacat.shema.converters;
+
+import com.netflix.metacat.common.QualifiedName;
+import com.netflix.metacat.common.dto.DatabaseDto;
+import com.netflix.metacat.common.dto.PartitionDto;
+import com.netflix.metacat.common.dto.TableDto;
+import org.apache.hadoop.hive.metastore.api.Database;
+import org.apache.hadoop.hive.metastore.api.Partition;
+import org.apache.hadoop.hive.metastore.api.Table;
+
+import java.util.List;
+
+/**
+ * Class to convert hive schema to metacat DTO and vice versa.
+ */
+public interface HiveMetacatConverters {
```

Comment:
Again to me this seems like a module that should be outside the core. We need a very generic interface... @ajoymajumdar might have different opinion but I think all the type converters should implement a common interface and we shouldn't have specific interfaces for hive, pig, etc. just implementations of the common interfaces